app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -2,17 +2,121 @@
 <layout
   xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
-  <androidx.constraintlayout.widget.ConstraintLayout
+  <data>
+    <variable
+      name="presenter"
+      type="org.oppia.app.topic.overview.TopicOverviewFragmentPresenter"/>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.topic.overview.TopicOverviewViewModel"/>
+  </data>
+  <ScrollView
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <TextView
-      android:id="@+id/dummy_text_view"
-      android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:overScrollMode="never"
+    android:scrollbars="none">
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:text="This is dummy TextView for testing"
-      app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent"/>
-  </androidx.constraintlayout.widget.ConstraintLayout>
+      android:paddingBottom="24dp">
+      <ImageView
+        android:id="@+id/topic_thumbnail_image_view"
+        android:layout_width="0dp"
+        android:layout_height="200dp"
+        android:src="@{viewModel.topic.topicThumbnail.thumbnailGraphicValue}"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"/>
+      <TextView
+        android:id="@+id/topic_name_text_view"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="24dp"
+        android:layout_marginEnd="24dp"
+        android:gravity="start"
+        android:text="@{viewModel.topic.name}"
+        android:textColor="@color/oppiaPrimaryText"
+        android:textSize="24sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/topic_thumbnail_image_view"/>
+      <TextView
+        android:id="@+id/story_count_text_view"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="24dp"
+        android:gravity="start"
+        android:text="@{viewModel.topic.storyCount + ` Stories`}"
```

Comment:
Please use a plural string for this instead of concatenation to ensure that translation can happen properly. See: https://github.com/oppia/oppia-android/pull/195#discussion_r337336255 for another comment I left recently that provides context on using plurals with data binding.

Reply:
Implemented plural-strings. This implementation is awesome and makes everything easy.
@BenHenning Thanks.